### DISCUSSION TITLE
Make deleting multiple backups easier

### DISCUSSION THREAD

**Discussion by pjannink** [home-assistant/frontend#21108] [Other feature requests]
Can you guys add selection of multiple backups to the SYSTEM/BACKUPS option? This to enable easier deletion of multiple backups. Basically same function as in fi Entities list where we can select multiple entities and use "Remove selected"

**Comment by ChristophCaina** [comment]
This is already available :)

![grafik](https://github.com/home-assistant/frontend/assets/26391061/3107041e-b060-4608-b495-8c97fc09d314)

**Comment by pjannink** [comment]
That is true for the full HAOS version ( I run a test instance in a VM on my laptop), but not for my production instance running in a container on Synology:

![image](https://github.com/home-assistant/frontend/assets/85283776/07979e4b-6f76-462e-a14c-6373dd019aa5)

**Comment by ChristophCaina** [comment]
ok, that was some missing information :)
Can you maybe share some information about the frontend version, and if you have a supervisor in your container (I don't have such a setup for ages, so I am not sure) which version supervisor is?

It might be not a frontend topic, if the backend doesn't support this option)

**Comment by pjannink** [comment]
My frontend version is 20240610.1. There is no supervisor as far as I know. I can not install add-ons, only HACS.
May be the request should be directed to the backend team? I see no real reason why multiple backup deletion should not be possible in HA running in a container
